crates/fluvio-cluster/src/start/k8.rs

```diff
@@ -850,9 +878,44 @@ impl ClusterInstaller {
         Ok(())
     }
 
-    /// Looks up the external address of a Fluvio SC instance in the given namespac
+    /// Uses kubectl port-forward to communicate with SC
+    #[instrument(skip(self, service))]
+    async fn start_sc_port_forwarding(
+        &self,
+        service: &K8Obj<ServiceSpec>,
+    ) -> Result<(String, u16, Child), K8InstallError> {
+        let pf_host_name = "localhost";
+
+        let target_port = ClusterInstaller::target_port_for_service(service)?;
+
+        // Wait for pod to start
+        sleep(Duration::from_secs(3)).await;
```

Comment:
This is fragile since it depends on arbitrary timing but let's defer that.  Let's add some notes in the code that this should be made more reliably retrying

Reply:
done